java/ingest/ingest-runner/src/main/java/sleeper/ingest/runner/impl/IngestCoordinator.java

```diff
@@ -432,6 +436,17 @@ public Builder<T> iteratorConfig(String iteratorConfig) {
             return this;
         }
 
+        /**
+         * The Sleeper filters configuration.
+         *
+         * @param  filtersConfig the configuration
+         * @return               the builder for call chaining
+         */
+        public Builder<T> filtersConfig(String filtersConfig) {
```

Comment:
It doesn't look like this is tested.

This should be populated automatically based on the table property, and it doesn't look like that will happen right now.

IngestRowsIT has shouldApplyIterator, we could add something similar. We can test that the filtering actually happens during ingest, based on the table property.